src/utils/updaters/removeEventElems.js

```diff
@@ -0,0 +1,7 @@
+export function removeEventElems() {
+	const eventElems = document.getElementsByClassName("event");
+	while (eventElems.length > 0) {
+		const eventElem = eventElems[0];
+		eventElem.parentNode.removeChild(eventElem);
```

Comment:
I can't understand how removing child from parentNode also removes it from eventElems array